### ISSUE TITLE
Rules, `selector` not respected

### ISSUE THREAD

**Issue by fredboyle** [felixzapata/gulp-axe-webdriver#26]
Attempting to limit which elements certain rules run against but not having any luck in my initial attempts.

```
a11yCheckOptions: {
      rules: {
        bypass: { enabled: false },
        'landmark-one-main': { enabled: false },
        region: { enabled: false },
        'page-has-heading-one': { enabled: false },
        'html-has-lang': { enabled: false },
        'scrollable-region-focusable': {
          'selector': ":not([class*='dropdown_length-'])"
        }
      },
```

What am I doing incorrectly? 

I've also tried with a bogus selector such as "p" but no change in results. The 'scrollable-region-focusable' rule is still run against other elements such as "div".

Thanks for your help.

**Comment by felixzapata** [comment]
hi, I wil taike a look ASAP

**Comment by felixzapata** [comment]
@fredboyle what is the rule that is not working? the rule with the selector?

**Comment by fredboyle** [comment]
@felixzapata Correct. The rule with the selector is not limiting elements tested at all, no matter what I place in the `selector` key.

**Comment by felixzapata** [comment]
I have to do more tests but maybe the problem is not related with the `gulp` plugin. I've checked that when you pass some rules to false (for example `'image-alt'`), the plugin is working but no with a selector. The system to set the rules is the same (there is no difference when you pass a selector or other value, it is just an object)

I've also checked updating the `axe-core` and `axe-webdriverjs` with the same results. It is works if you disable rules but no with selector.

So I don't know if the problem is because the `axe-core` doesn't have examples related with selectors (just only how to) and we are not pass the value correctly or maybe the problem is a bug of `axe-core` itself.
